DirectEmployers/MyJobs
# [PD 2606] Add Autofocus to Tag widgets

## What's this PR do?
This PR adds autofocus to the tags widget inside of NUO. It also allows you to press the enter key and add a new tag when someone wants to create one

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of NUO widget and create or edit a partner. Click on the tags section and bring up the tags widget

## How should this be manually tested?
Pull down the repository. Go inside of the NUO app and edit or create a new partner or contact. When you click to edit the tags section you should see the focus go into the add new tag box. Once one has been created and added, focus will go back into the add a new tag box. Along with this, instead of having to click on the add new button to add a new tag, you will also be able to click the enter key and have it add a new tag as well

## Any background context you want to provide?
This also fixed the problem with state mutation being had when it comes to adding a new tag or removing an existing tag
